h2o-genmodel/src/main/java/hex/genmodel/algos/tree/ContributionsPredictor.java

```diff
@@ -1,12 +1,16 @@
 package hex.genmodel.algos.tree;
 
 import hex.genmodel.PredictContributions;
+import hex.genmodel.attributes.parameters.FeatureContribution;
 import hex.genmodel.utils.ArrayUtils;
 
+import java.util.Arrays;
+
 public abstract class ContributionsPredictor<E> implements PredictContributions {
   private final int _ncontribs;
   private final String[] _contribution_names;
   private final TreeSHAPPredictor<E> _treeSHAPPredictor;
+  private final ContributionComposer contributionComposer = new ContributionComposer();
```

Comment:
Does this need to be instance variable? It doesn't add any value if it is right now - if it wasn't it wouldn't need to be Serializable - one less problem to worry about.

Reply:
True, it can be only instance or the methods could be static, maybe it is the best solution

Reply:
I made it only instance variable